gaas-ui/.stylelintignore

```diff
@@ -0,0 +1,3 @@
+/coverage
+
+/build
```

Comment:
Not to be too annoying but no newline at end of file

Reply:
No problem

Reply:
Done